README.md

```diff
@@ -1,464 +1,63 @@
-# Environment Variables
 
-| ENV | Values | Description |
-| --------------------------- | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| MODE | DEV, PROD | This is the mode of running the application. There are 2 modes available. DEV for the Development mode and PROD for the production mode. Certain settings will be changed according to the mode. Such as in the PROD mode, mutation will run asynchronously and synchronously otherwise. Same applies to DEBUG, it will be OFF in PROD and TRUE otherwise. |
```

Comment:
@bereketassefa You think the existing repo is not important.

Reply:
please do that and make sure you create a separate commit for the documentation and for the json changes.